### DISCUSSION TITLE
After update lazyvim, My rust codes  missed inlay hints

### DISCUSSION THREAD

**Discussion by golovo** [LazyVim/LazyVim#2843] [Q&A]
How could I enable inlay hints in rust now ?   it is default enable inlay hints with lazyvim older version.
Thanks a lot!

**Comment by folke** [comment]
It seems that rustacean removed the inlay hints code in favor of the native inlay hints on 0.10.0

To get the native inlay-hints on 0.10.0, you can enable them in your config.
https://www.lazyvim.org/plugins/lsp#nvim-lspconfig

**Reply by golovo** [reply]
Thanks for your reply and awesome works.  
I had tried add some lua files to lua/plugin dir for lsp and rustaceanvim([https://github.com/mrcjkb/rustaceanvim/discussions/46#discussioncomment-7620822]) . But both were not work.
Maybe I should wait fedora to update neovim to >=0.10 version.
